### ISSUE TITLE
All Give admin pages should have a valid H1 

### ISSUE THREAD

**Issue by DevinWalker** [WordImpress/Give#2047]
## Issue Overview
Currently some settings pages do not have an h1 tag and it's bad for a11y and also user experience: 

![2017-09-06_21-15-30](https://user-images.githubusercontent.com/1571635/30145798-bb233108-9348-11e7-853b-3bc9194258dd.gif)

We need to add valid H1 titles to each settings page. It should match the title in the menu. 

1. Settings > Settings
2. Tools > Tools
3. Reports > Reports

## Expected Behavior
All menu items have similar h1s

## Current Behavior
Some do, some don't... inconsistent 

## Possible Solution
Make consistent
